config.yaml

```diff
@@ -55,6 +55,12 @@ options:
       applications from a wide range of attacks, including the OWASP Top Ten, with a minimum of
       false alerts. See https://github.com/coreruleset/coreruleset for more details.
     type: boolean
+  owasp-modsecurity-custom-rules:
+    default: ""
+    description: >
+      \n separated list of custom rules to be added to modsecurity-snippet annotation. 
```

Comment:
why is there an \n at first in the description?

Reply:
Also looks like there could be trailing whitespace?

Reply:
Sorry, when I wrote made sense in my head. It's a list separated by "\n" like
```'SecAction "id:900130,phase:1,nolog,pass,t:none,setvar:tx.crs_exclusions_wordpress=1"\nSecAction "id:900130,phase:1,nolog,pass,t:none,setvar:tx.crs_exclusions_drupal=1"\n'```

Reply:
Trailing whitespace removed.

Reply:
Do you think would be better to use another separator or just change the description to make it clear that '\n' is the separator? Using \n is easier because it's not necessary to format the config and I think would be safe to use with [SecRule/SecAction configs](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#secrule)

Reply:
Oh I see now, maybe quote or describe it `new line ('\n') separated ...`?